Wireframe/css/style.css

```diff
@@ -0,0 +1,164 @@
+:root {
+  --font-size: sans-serif;
+  --dark-orange: #c05326;
+  --light-gray: #dad7cd;
+  --slate-dark: #4c5760;
+  --light-blue: #bcb8b1;
+}
+
+body {
+  color: var(--slate-dark);
+  margin: 5px;
+  font-size: 18px;
+  background-color: rgb(244, 244, 240);
+  font-family: sans-serif;
+}
+
+.gitImage {
+  display: grid;
+  height: 150px;
+  width: 150px;
+  justify-content: center;
+  align-content: center;
+  border-radius: 75px;
+  opacity: 0.7;
+}
+
+.gridContainer {
+  height: 100%;
+  display: grid;
+  grid-gap: 20px;
+  padding: 20px;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 200px 400px 300px 80px;
```

Comment:
Avoid sizing heights directly, especially to contain text. In real life, you generally can't control the text you are flowing into boxes, as your users are entering this. For this reason, practice the habit of describing relationships and behaviours in your code, not drawing fixed boxes. 

Remember the mug: https://www.amazon.co.uk/CSS-is-Awesome-Printed-Mug/dp/B00NP25SY8